pdr_backend/classifiermodel/test/test_classifiermodel_data_factory.py

```diff
@@ -0,0 +1,388 @@
+import numpy as np
```

Comment:
Small feedback: this file can probably be 1/2 page long, since ClassifierModelDataFactory mostly reuses RegressionModelDataFactory. Will help maintenance, a lot.